pine-a64-plus/armbian.yaml

```diff
@@ -0,0 +1,93 @@
+
+{{- $image := or .image "debos-debian-bullseye-pine-a64.img" -}}
+
+architecture: "arm64"
+
+
+actions:
+  - action: debootstrap
+    suite: "bullseye"
+    components:
+      - main
+      - contrib
+      - non-free
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    description: Install some packages
+    packages: 
+        - sudo
+        - openssh-server
+        - isc-dhcp-client
+        - iproute2
+        - ifupdown
+
+# Armbian-specific (kernel, bootloader and settings)
+
+  - action: overlay
+    source: overlays/armbian
+
+  - action: apt
+    description: Kernel, U-Boot, DeviceTree install for Pine A64 (armbian)
+    packages: 
+        - armbian-firmware
+        - linux-u-boot-pine64-current
+        - linux-image-current-sunxi64
+        - linux-dtb-current-sunxi64
+        - sunxi-tools 
+        - armbian-bsp-cli-pinebook-a64
+        - init
+        - initramfs-tools
+        
+  - action: run
+    description: Generate initramfs
+    chroot: true
+    command: update-initramfs -k all -c 
+
+# End of Armbian-specific
+
+  - action: run
+    chroot: true
+    command: echo debos-debian-pine-a64 > /etc/hostname
+
+  - action: run
+    description: Set up user
+    chroot: true
+    script: scripts/setup-user.sh
+
+  - action: run
+    description: Network setup
+    chroot: true
+    script: scripts/setup-network.sh
+
+  - action: image-partition
+    imagename: {{ $image }}
+    imagesize: 1GB
+    partitiontype: msdos
+    mountpoints:
+      - mountpoint: /
+        partition: root
+    partitions:
+      - name: root
+        fs: ext4
+        start: 1% 
+        end: 100%
+        flags: [ boot ]
+        fsuuid: 0129b70b-dfe6-4c6c-9fa1-8683a2f2dfdb
+  
+
+  - action: filesystem-deploy
+    description: Deploying filesystem onto image
+
+
+  - action: run
+    description: Install U-Boot bootloader
+    postprocess: true
+    command: TARGET="/usr/lib/u-boot/pine64_plus/" u-boot-install-sunxi64 {{ $image }}
+
+  - action: run
+    description: Pack result image
+    postprocess: true
+    command: xz -z -v -9 -T 4 --keep --force {{ $image }}
```

Comment:
Please use `-T 0` and remove `--keep --force`.
Or, if you want to be more flexible -- use variables in recipe file, so we can redefine them via command line, for instance:
```
{{ $xzopts := or .xzopts "--verbose" }}
...
    command: xz -z -9 -T {{ $parallel }} {{ $xzopts }} {{ $image }}
```
then call the `debos` with:
 `debos -t xzopts:"--keep --force --verbose" ....`

Reply:
FYI I found zstd to be the best-performing compressor with my images, both in terms of speed and compression ratio.

Reply:
Last time I just experimenting much with xz, so let it be xz